BitmarkRegistry/AppDelegate.swift

```diff
@@ -35,6 +35,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     evaluatePolicyWhenUserSetEnable()
     initSentry()
     Global.log.logAppDetails()
+    
+    // Register APNS
+    UIApplication.shared.registerForRemoteNotifications()
```

Comment:
I think in this case we can miss register APNS Token into our server, because evaluatePolicyWhenUserSetEnable is called before register; and also we are unsure when the APNS token is registered that can make registerAPNS() do nothing, right?

Reply:
Right, I'll update.